src/protocols/rdp/rdp_settings.h

```diff
@@ -359,6 +359,14 @@ typedef struct guac_rdp_settings {
      * the destination directory is otherwise ambiguous).
      */
     char* sftp_directory;
+
+    /**
+     * The interval at which SSH keepalive messages are sent to the server for
+     * SFTP connections.  The default is 0 (disabling keepalives), and a value
+     * of 1 is automatically increased to 2 by libssh2 to avoid busy loop corner
+     * cases.
+     */
+    unsigned sftp_server_alive_interval;
```

Comment:
Using an `unsigned int` for the result of `guac_user_parse_args_int()` is dangerous. Because `guac_user_parse_args_int()` accepts negative values, such values will be assigned directly without any warning and interpreted as enormous positive values.

If you want to disallow negative values, it would be better to explicitly check for those values and round up to a legal value as necessary.